### DISCUSSION TITLE
How to DELETE Nhost account?

### DISCUSSION THREAD

**Discussion by PtradeLLC** [nhost/nhost#2363] [Q&A]
Hi all,
Would like to know how to delete Nhost account from `app.nhost.io` - Using Nhost is not a good fit for what we needed, our use case, and would like to delete our account all together.

We have deleted `Project` and `Workspace` with the assumption that this is going to lead to deleting the account all together, but the user account still persists with no way to delete. Any pointers helps as we don't want unsolicited email/newsletter that we can't get out of.

Thanks

**Comment by dbarrosop** [comment]
Give us a couple of days, we are actually about to release a new dashboard version that should allow you to delete your account yourself.

> not a good fit for what we needed

If there is any feedback you'd like to share that'd be appreciated.

**Comment by dbarrosop** [comment]
You should be able to do this now yourself under "Account Settings"
